src/test/java/org/openrewrite/launchdarkly/ChangeVariationDefaultTest.java

```diff
@@ -0,0 +1,220 @@
+        void changeDefaultValueToTrueEvenIfVariable() {
+            rewriteRun(
+              // language=java
+              java(
+                """
+                  import com.launchdarkly.sdk.LDContext;
+                  import com.launchdarkly.sdk.server.LDClient;
+                  class Foo {
+                      private LDClient client = new LDClient("sdk-key-123abc");
+                      void bar(LDContext context) {
+                          boolean defaultValue = false; // Not cleaned up
+                          if (client.boolVariation("flag-key-123abc", context, defaultValue)) {
+                              System.out.println("Feature is on");
+                          }
+                      }
+                  }
+                  """,
+                """
+                  import com.launchdarkly.sdk.LDContext;
+                  import com.launchdarkly.sdk.server.LDClient;
+                  class Foo {
+                      private LDClient client = new LDClient("sdk-key-123abc");
+                      void bar(LDContext context) {
+                          boolean defaultValue = false; // Not cleaned up
+                          if (client.boolVariation("flag-key-123abc", context, true)) {
+                              System.out.println("Feature is on");
+                          }
+                      }
+                  }
+                  """
+              )
+            );
+        }
+    }
+
+    @Nested
+    class StringVariation {
+        @Test
+        @DocumentExample
+        void changeDefaultValueToTrue() {
+            rewriteRun(
+              // language=java
+              java(
+                """
+                  import com.launchdarkly.sdk.LDContext;
+                  import com.launchdarkly.sdk.server.LDClient;
+                  class Foo {
+                      private LDClient client = new LDClient("sdk-key-123abc");
+                      void bar(LDContext context) {
+                          if (client.stringVariation("flag-key-123abc", context, "foo")) {
+                              System.out.println("Feature is on");
+                          }
+                      }
+                  }
+                  """,
+                """
+                  import com.launchdarkly.sdk.LDContext;
+                  import com.launchdarkly.sdk.server.LDClient;
+                  class Foo {
+                      private LDClient client = new LDClient("sdk-key-123abc");
+                      void bar(LDContext context) {
+                          if (client.stringVariation("flag-key-123abc", context, "true")) {
+                              System.out.println("Feature is on");
+                          }
+                      }
+                  }
+                  """
+              )
+            );
+        }
+    }
+
+    @Nested
+    class DoubleVariation {
+        @Test
+        @DocumentExample
+        void changeDefaultValueToTrue() {
```

Comment:
Stylistic: the test name here doesn't really match up with the test content.

Reply:
```suggestion
        void changeDefaultValueTo456() {
```